test_models/valids/annotations.json

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
I created a new example model just for testing the transformation related to annotations.

Reply:
Done